_posts/0000-01-03-dumpsterfire.md

```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Happy Thanksgiving! 
```

Comment:
Although this is inline with the image, maybe we could try to be funnier? 

@ericalloyd to get your opinion on this

Reply:
Yea, I'll definitely fix that, good idea! 💡